src/linter/linter.ts

```diff
@@ -8,14 +8,43 @@ import posixPath from "node:path/posix";
 import {stat} from "node:fs/promises";
 import {ProjectGraph} from "@ui5/project";
 import {AbstractReader} from "@ui5/fs";
+import ConfigManager from "../utils/ConfigManager.js";
+import {Minimatch} from "minimatch";
 
 async function lint(
 	resourceReader: AbstractReader, options: LinterOptions
 ): Promise<LintResult[]> {
 	const lintEnd = taskStart("Linting");
+	let {ignorePattern} = options;
 
-	const workspace = createWorkspace({
+	if (!ignorePattern) {
+		const configMngr = new ConfigManager();
+		const config = await configMngr.getConfiguration();
+		ignorePattern = config.flatMap((item) => item.ignores).filter(($) => $) as string[];
+	}
+
+	const miniChecks = ignorePattern.map((ignore) => new Minimatch(ignore));
+
+	const filteredCollection = createFilterReader({
 		reader: resourceReader,
+		callback: (resource) => {
+			if (!miniChecks?.length) {
+				return true;
+			}
+
+			// Minimatch works with FS and relative paths.
+			// So, we need to convert virtual paths to absolute
+			// FS paths and strip the rootDir
+			const resPath = transformVirtualPathToFilePath(
+				resource.getPath(), options.rootDir, "/resources", "test", "/test-resources")
+				.replace(options.rootDir, "");
+
+			return miniChecks.some((check) => !check.match(resPath, true));
```

Comment:
Looks like we missed to discuss this as part of CPOUI5FOUNDATION-782: What about un-ignoring files? (compare to https://eslint.org/docs/latest/use/configure/ignore#unignoring-files-and-directories)

Reply:
Now it can ignore & unignore files.